File: ejercicios/python/ejercicio042.py
media = 0
total = 0
num = -1

while num != 0:
    num = float(input("Ingrese un nœmero (0 para finalizar): "))
    media = media + num
    total = total + 1

if total != 1:
    media = media / (total - 1)
    print("El promedio de los " + str(total - 1) + " nœmeros ingresados es: " + str(media))
else:
    print("No se ingresaron nœmeros. ")

File: ejercicios/c++/ejercicio042.cpp
#include<iostream>

using namespace std;

int main() {

	double media = 0;
	double num;
	int total=0;

	do {
		cout << "Ingrese un nœmero (0 para finalizar): ";
		cin >> num;
		media = media + num;
		total = total + 1;
	} while (num != 0);
	
	if( total != 1){	
		media = media/(total-1);
		cout << "El promedio de los " << (total - 1) << " nœmeros ingresados es: " << media << endl;
	}
	else{	
	cout << "No se ingresaron nœmeros." << endl;
	}
	return 0;
}

